sim/dex-formats.ts

```diff
@@ -15,7 +15,7 @@ export type ModdedFormatData = FormatData | Omit<FormatData, 'name'> & {inherit:
 export interface FormatDataTable {[id: IDEntry]: FormatData}
 export interface ModdedFormatDataTable {[id: IDEntry]: ModdedFormatData}
 
-type FormatEffectType = 'Format' | 'Ruleset' | 'Rule' | 'ValidatorRule';
+type FormatEffectType = 'Format' | 'Rule' | 'ValidatorRule';
```

Comment:
Oh nice I was going to ask if Ruleset was actually used anywhere and I'm pleased to learn the answer is no.